terminus-ui/input/src/input.component.ts

```diff
@@ -1427,7 +1427,6 @@ export class TsInputComponent implements
     } else {
       const finalValue = !this.maskSanitizeValue ? value : this.cleanValue(value, this.currentMask.unmaskRegex);
       this.onChangeCallback(finalValue);
-      this.inputValueAccessor.value = finalValue;
```

Comment:
why would this remove the % symbol?

Reply:
From my current understanding, assigning back to the CVA here triggers Angular's internal updates which then overwrite the UI value with the model value.

Reply:
Weird